targets/TARGET_ONSEMI/TARGET_NCS36510/ncs36510_user_trim.json

```diff
@@ -0,0 +1,8 @@
+{
+  "mac-addr-low": "0x12345678",
```

Comment:
I wonder if this should be part of the targets.json and add this config there or placed here as it is

cc @sg- @bogdanm @c1728p9